components/zcash_address/src/lib.rs

```diff
@@ -146,9 +145,8 @@ mod kind;
 #[cfg(any(test, feature = "test-dependencies"))]
 pub mod test_vectors;
 
-pub use convert::{
-    ConversionError, Converter, ToAddress, TryFromAddress, TryFromRawAddress, UnsupportedAddress,
-};
+use convert::Converter;
```

Comment:
Why is `Converter` being removed from the public API? Is this a rebase conflict resolution bug?

Reply:
Oops, yes.